cmake/remillConfig.cmake.in

```diff
@@ -26,13 +26,13 @@ foreach(_comp ${remill_FIND_COMPONENTS})
 
   if ("VCPKG_DEPS" STREQUAL _comp)
     set(remill_setup_vcpkg True)
-    if(NOT "x@VCPKG_ROOT@x" STREQUAL "xx" AND NOT USE_SYSTEM_DEPENDENCIES)
-      if (EXISTS "@VCPKG_ROOT@")
-        set(VCPKG_ROOT "@VCPKG_ROOT@"
-          CACHE PATH "Location of dependency libraries"
+    if(NOT "x@CMAKE_TOOLCHAIN_FILE@x" STREQUAL "xx" AND NOT USE_SYSTEM_DEPENDENCIES)
+      if (EXISTS "@CMAKE_TOOLCHAIN_FILE@")
+        set(CMAKE_TOOLCHAIN_FILE "@CMAKE_TOOLCHAIN_FILE@"
+          CACHE PATH "Location of CMake toolchain file"
           )
-        include(${CMAKE_CURRENT_LIST_DIR}/vcpkg_helper.cmake)
-        message(STATUS "Found VCPKG_ROOT: ${VCPKG_ROOT}")
+        include(${CMAKE_TOOLCHAIN_FILE})
+        message(STATUS "Found CMAKE_TOOLCHAIN_FILE: ${CMAKE_TOOLCHAIN_FILE}")
         set(vcpkgdeps_FOUND)
       endif()
     endif()
```

Comment:
I think we should look into getting rid of most vcpkg references in this file. We should only be listing `find_dependency` things, so the whole file would look like this:

```cmake
set(REMILL_LLVM_VERSION @LLVM_MAJOR_VERSION@)

 # External libs
include(CMakeFindDependencyMacro)
find_dependency(XED)
find_dependency(glog)
find_dependency(Z3)
find_dependency(LLVM)

# Exported Targets
include("${CMAKE_CURRENT_LIST_DIR}/remillTargets.cmake")

if(NOT TARGET sleigh::support)
  find_dependency(sleigh COMPONENTS Support Specs)
endif()
```

I'm not sure if that will handle the sleigh dependency correctly. It'll depend on how sleigh targets are handled when using `fetchContent`.

Reply:
[76d234b](https://github.com/lifting-bits/remill/pull/639/commits/76d234bd40d53d7ed0f37473ea18d85ba9b37119)

I had to move the `find_dependency` for Sleigh before the `remillTargets.cmake` include but other than that, it seems to work ok with Anvill.